autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/feature_preprocessing/base_feature_preprocessor.py

```diff
@@ -8,7 +12,13 @@
 class autoPyTorchFeaturePreprocessingComponent(autoPyTorchTabularPreprocessingComponent):
     _required_properties: List[str] = ['handles_sparse']
 
-    def __init__(self) -> None:
+    def __init__(self, random_state: Optional[np.random.RandomState] = None) -> None:
```

Comment:
Once you set any kwargs, we do not need `-> None` for `__init__` function.
There are some that have `-> None`, so could you remove them?

Reply:
That would involve changing a lot of files through out the project which I don't think is in the scope of this PR

Reply:
I mean only part that comes up in your fix 
I didn't say everything

Reply:
Okay, I'll do that.